biopandas/pdb/pandas_pdb.py

```diff
@@ -303,6 +321,25 @@ def _fetch_pdb(pdb_code):
             print('URL Error %s' % e.args)
         return url, txt
 
+    @staticmethod
+    def _fetch_af2(uniprot_id: str):
+        """Load PDB file from https://alphafold.ebi.ac.uk/."""
+        txt = None
+        url = f"https://alphafold.ebi.ac.uk/files/AF-{uniprot_id.upper()}-F1-model_v2.pdb"
```

Comment:
The only thing to point out is that the `v2` in the URL probably shouldn't be hardcoded. Any ideas on how to manage this (maybe an arg in the `PandasPdb` init that defaults to 2?)